client/state/comments/actions.js

```diff
@@ -17,6 +17,7 @@ import {
 	COMMENTS_WRITE,
 	COMMENT_REQUEST,
 	COMMENTS_TREE_SITE_REQUEST,
+	COMMENTS_TREE_JETPACK_SITE_REQUEST,
```

Comment:
No need for the new request type or action. This is available as `COMMENTS_LIST_REQUEST` and `requestCommentsList()` respectively